src/App.js

```diff
@@ -41,7 +42,11 @@ const App = () => {
 			/>
 			<div class="counter-count">{count}</div>
 			<div class="button-container">
-				<button class="button-subtract" onClick={handleCountSubtract}>
+				<button
+					class="button-subtract"
```

Comment:
please add style to this class so that the user gets an understanding that the button is disabled